pkg/cloud/cloud.go

```diff
@@ -94,6 +96,8 @@ const (
 	volumeDetachedState = "detached"
 	volumeAttachedState = "attached"
 	cacheForgetDelay    = 1 * time.Hour
+
+	assumeRoleDuration = 1 * time.Hour
```

Comment:
what is the significance of this duration? what happens after one hour?

Reply:
The credentials sts vends is valid for an hour. After an hour it refreshes. Default is 15 mins. This is what was done for Karpenter so I used it, happy to remove it.

Reply:
Thanks for explaining! - i see we are referencing ENO from https://code.amazon.com/packages/EKSNodeOperator/blobs/9702a7dd7e23e968170f46a3eabe44b92128635a/--/pkg/awsclients/provider.go#L14,L47,L48,L49, consider naming this as `sessionDuration`, that might make more sense.


just to confirm - with this implementation, after the session expires in an hour, the assume role provider would be able to assume the role again?

Reply:
Yes, that is what I've been informed by my team